apiserver/admin.go

```diff
@@ -156,16 +157,15 @@ func (a *admin) login(req params.LoginRequest, loginVersion int) (params.LoginRe
 	}, nil
 }
 
-func (a *admin) getAuditRecorder(req params.LoginRequest, authResult *authResult) (*auditlog.Recorder, error) {
-	if !authResult.userLogin || a.srv.auditLogger == nil {
+func (a *admin) getAuditRecorder(req params.LoginRequest, authResult *authResult, cfg auditlog.Config) (*auditlog.Recorder, error) {
+	if !authResult.userLogin || cfg.Enabled == false {
```

Comment:
don't need cfg.Enabled == false, just !cfg.Enabled

Reply:
Oh duh, I hate it when other people do that!